_includes/header.html

```diff
@@ -1,35 +1,38 @@
+<a href="#main-content" class="screen-reader-text focusable">Skip to content</a>
 <header id="page-header" role="banner">
 	<div class="page-header-content">
-		<a href="http://10up.com" class="logo-link">10up</a>
-		<nav class="page-nav top-nav" role="navigation" aria-label="main-navigation">
-			<ul class="header-nav nav-dropdowns js-mobile-expandable">
+		<nav class="site-navigation" role="navigation" itemscope="itemscope" itemtype="http://schema.org/SiteNavigationElement">
+
+			<a href="#primary-nav" aria-controls="primary-nav" class="site-menu-toggle toggle-menu-icon">
+				<span class="screen-reader-text">Primary Menu</span>
+			</a>
+
+			<ul id="primary-nav" class="primary-menu">
 				{% for category in site.data.categories %}
 					{% if page.category == category.slug %}
-					<li class="current-menu-item">
+					<li class="menu-item current-menu-item">
 					{% else %}
-					<li>
+					<li class="menu-item menu-item-has-children">
 					{% endif %}
 						<a href="#">{{ category.slug }}</a>
-						<ul class="subnav">
+						<ul class="sub-menu" aria-hidden="true">
```

Comment:
@kdo can we add these hiddens with JS so if it doesn't load they're not permanently hidden from screen readers?

Reply:
@timwright12 thanks for catching this! removed, it's already taken care of in the JS by our plugin